maine-thesis.cls

```diff
@@ -859,7 +859,12 @@
 	\vfill
 	\clearpage
 }
-
+\newcommand{\glsa}{%
```

Comment:
Can you document this command in the Documentation (Ch3.tex, which corresponds to Chapter 4 Front.tex)?  Please include any guidelines for where to place it and whether the Graduate School requires it or considers it optional.

Reply:
TeX command is now added to the documentation with appropriate placement suggestions. The Graduate School has it in its downloadable template but does not mention it in its guideline PDF.